pkg/virt-operator/resource/generate/components/secrets.go

```diff
@@ -44,10 +46,10 @@ var populationStrategy = map[string]CertificateCreationCallback{
 		}
 		keyPair, _ := triple.NewServerKeyPair(
 			caKeyPair,
-			VirtOperatorServiceName+"."+secret.Namespace+".pod.cluster.local",
+			VirtOperatorServiceName+"."+secret.Namespace+caPodClusterLocal,
```

Comment:
we add the "." between virt operator and secret namespace, but then the "." is implied with caPodClusterLocal. let's make this consistent.